areas.html

```diff
@@ -139,47 +145,48 @@
       type="text/css"
     />
     <link rel="stylesheet" href="map_with_sidebar.css" type="text/css" />
-    <style>
-      .mapboxgl-popup {
-        width: 600px;
-        /* TODO Height seems responsive to content, can't make it take lots of the screen */
-      }
-    </style>
   </head>
 
   <body>
-    <div id="map">
-      <div id="left" class="sidebar flex-center left">
-        <div class="sidebar-content rounded-rect flex-center">
+    <aside id="sidebar">
+      <div class="sidebar-content content-container">
+        <nav>
           <a href="index.html">Home</a>
-          <div id="app_nav"></div>
-          <h2>Area Interventions</h2>
-          <p>
-            Draw the boundaries by clicking on draw button (top right of the
-            map). Click on the first point of when the boundary is complete and
-            add data fields.
-          </p>
-          <h1 id="authority"></h1>
-          <p>Population: <span id="population"></span></p>
-          <p>Number of areas drawn: <span id="num_areas">0</span></p>
+          <a href="areas.html" class="authority-link">Areas</a>
```

Comment:
It's much nicer to modify these in place, instead of inserting everything from JS